docarray/typing/text.py

```diff
@@ -0,0 +1,33 @@
+import string
+from typing import Type
+
+from docarray.document.base_node import BaseNode
+from docarray.proto import NodeProto
+
+
+class Text(BaseNode):
```

Comment:
I don't think we want this as a type, `Text` is supposed to be a pre-defined Document, not a type